source-azure-blob-storage/test.flow.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
From what I understand this isn't currently working, so we can get rid of it here.